styles.css

```diff
@@ -21,6 +21,10 @@ body {
 	z-index: 10;
 }
 
+#noscript h1 {
+	font-size: 1.8em;
```

Comment:
Any particular reasoning why? I'm not sure it matters how this part looks so long as it's readable and tells you how to fix the issue